docs/cli_commands.md

```diff
@@ -247,7 +247,7 @@ minikube service [command]
 
 ---
 ### start
-**Description -** Starts a local kubernetes cluster using VM. This command assumes you have already installed one of the VM drivers: **virtualbox/parallels/vmwarefusion/kvm/xhyve/hyperv**.
+**Description -** Starts a local kubernetes cluster using VM. This command assumes you have already installed one of the VM drivers: **virtualbox/parallels/vmwarefusion/kvm/hyperv**.

```

Comment:
kvm2 ?

Reply:
Maybe something like: `fmt.Sprintf("....%v", constants.SupportedVMDrivers))`. Then once the PR https://github.com/kubernetes/minikube/pull/4738 is merged, it would list the correct drivers for each platform. 

wdyt?

Reply:
Removed.